src/excmds.ts

```diff
@@ -3446,6 +3446,7 @@ import * as hinting from "@src/content/hinting"
         - `-pipe selector key` e.g, `-pipe a href` returns the key. Only makes sense with `composite`, e.g, `composite hint -pipe * textContent | yank`. If you don't select a hint (i.e. press <Esc>), will return an empty string.
         - `-W excmd...` append hint href to excmd and execute, e.g, `hint -W exclaim mpv` to open YouTube videos.
         - -q* quick (or rapid) hints mode. Stay in hint mode until you press <Esc>, e.g. `:hint -qb` to open multiple hints in the background or `:hint -qW excmd` to execute excmd once for each hint. This will return an array containing all elements or the result of executed functions (e.g. `hint -qpipe a href` will return an array of links).
+        - -j* disable javascript hints. Don't generate hints related to javascript events. This is particularly useful when used with the `-c` option when you want to generate only hints for the specified css selectors. Also useful on sites with plenty of useless javascript elements such as google.com
         - -br deprecated, use `-qb` instead
```

Comment:
Please add -jc.

Also, I think it makes more sense that j would enable rather than disable JS hints.

I think the usual convention is that uppercase letters disable an option, so -J might make more sense?

Reply:
Let's have it include JS hints by default (this is the current behaviour, right?).

Your code seems to preclude, e.g. `-qjc`. Is that on purpose?

Reply:
Ok, I will do that.

What do you mean preclude? -qjc works just as -qc minus the javascript hints. It doesn't prevent the use of other options or anything like that. Did you mean precede? Not sure what you are referring to there.